qiskit/opflow/gradients/circuit_gradients/param_shift.py

```diff
@@ -194,7 +197,7 @@ def _parameter_shift(
             if self.analytic:
                 # Unroll the circuit into a gate set for which the gradient may be computed
                 # using pi/2 shifts.
-                circ = ParamShift._unroll_to_supported_operations(circ)
+                circ = self._transpile_to_supported_operations(circ, self.SUPPORTED_GATES)
                 operator = ParamShift._replace_operator_circuit(operator, circ)
```

Comment:
I saw the comment in the CircuitGradient subclass of using self to access the staticmethod of the parent. From a presonal preference I usually do it by classname so it differentiates it. In this case here you also changed it - now it ends up being back to back with another call done the other way so things look inconsistent now,

Reply:
@woodsp-ibm You are right, I did not notice that there are numerous calls in the class that use a class name to call static methods. I tried to read some discussions about using self vs class name in such cases to get a better context and there seems to be no difference for our case (there is a difference if both parent and child classes implement a method with the same name because self might evaluate to one or another depending on the context). 

I would prefer having a consistent use here and since there are other calls in the class that use a class name, I would be inclined to actually revert my change. 

Since @mtreinish suggested that his comments regarding that are not critical, I suppose he would not mind?